.github/workflows/pr.yaml

```diff
@@ -67,6 +69,49 @@ env:
     CONTAINERS_DATA_SOURCE
     PROCTREE_DATA_SOURCE
 jobs:
+  #
+  # DOC VERIFICATION
+  #
+  verify-docs:
+    name: Verify Documentation
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+      - name: Check if flags flags man/markdown docs were changed
+        id: changed-files
+        uses: tj-actions/changed-files@v40
+        with:
+          files_yaml: |
+            md_files:
+              - "docs/docs/flags/**/*.1.md"
+            man_files:
+              - "docs/man/**/*.1"
+      - name: Ensure updates of *.1.md and *.1 pairs
+        if: ${{ steps.changed-files.outputs.md_files_any_changed == 'true' || steps.changed-files.outputs.man_files_any_changed == 'true' }}
+        run: |
+          md_files=$(echo "${{ steps.changed-files.outputs.md_files_all_changed_files }}" | xargs -n 1 basename | sed 's/\.1.md$//')
+          man_files=$(echo "${{ steps.changed-files.outputs.man_files_all_changed_files }}" | xargs -n 1 basename | sed 's/\.1$//')
+
+          missing_updates=()
+          for base_name in $md_files; do
```

Comment:
The script assumes that for every .1.md file, there is a corresponding .1 file that needs to be updated, and vice versa. If this is not strict (like a new file added), the script may raise false positives, no ?

Reply:
Well observed. I need to change it to [any_modified](https://github.com/tj-actions/changed-files#output_any_modified) in the `if` and to [all_modified_files](https://github.com/tj-actions/changed-files#output_all_modified_files) in the assignments. Going to, hold on.

Reply:
Updated Makefile to ensure that any change (addition, modification, deletion) in the .md will reflect in the man pages.

Reply:
![image](https://github.com/aquasecurity/tracee/assets/3372117/3959a44d-25b8-421f-a707-45d7722991a4)

This error should be ignored, as it was a remnant caused precisely by the lack of checking (checking introduced by this PR).